src/logger.js

```diff
@@ -0,0 +1,55 @@
+/* eslint no-console:0 */
+
+import chalk from 'chalk';
+import {isFunction, isNil} from 'lodash';
+
+function log(message, userFn, colorFn) {
+  if(!isNil(userFn) && !isFunction(userFn)) {
```

Comment:
wouldn't logic be simpler if you didn't have to do the check on each log action and instead filled undefined methods in config with dummy console.log implementation?